File: src/main/java/carelog/carelog/relation/web/dto/RelationCreateRequest.java
package carelog.carelog.relation.web.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotNull;

import java.util.UUID;

public record RelationCreateRequest(
        @Schema(description = "고객 공개 ID")
        @NotNull(message = "고객 PublicId는 필수 입력 값입니다.")
        UUID customerPublicId
) {}

File: src/main/kotlin/carelog/carelog/relation/web/dto/RelationCreateRequest.kt
package carelog.carelog.relation.web.dto

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotNull
import java.util.UUID

data class RelationCreateRequest(
    @field:NotNull(message = "고객 PublicId는 필수 입력 값입니다.")
    @Schema(description = "고객 공개 ID")
    val customerPublicId: UUID,
)
